packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -197,7 +191,9 @@ const Button = ({
           data-se={testId}
           disabled={isDisabled}
           endIcon={endIcon}
-          fullWidth={isFullWidth}
+          fullWidth={
+            buttonContext.isFullWidth ? buttonContext.isFullWidth : isFullWidth
+          }
```

Comment:
Can't we do this instead?:
```suggestion
          fullWidth={
            buttonContext.isFullWidth ?? isFullWidth
          }
```